usr/share/rear/lib/layout-functions.sh

```diff
@@ -1264,4 +1264,3 @@ delete_dummy_partitions_and_resize_real_ones() {
     last_partition_number=0
 }
 
-# vim: set et ts=4 sw=4:
```

Comment:
This change does not belong to "support for s390 (zLinux) arch"
so that it should be removed from this pull request.